pipes/WDL/tasks/tasks_16S_amplicon.wdl

```diff
@@ -69,17 +79,45 @@ task trim_reads {
 
     input {
         File    reads_qza
-        #Boolean not_default = false
         String  forward_adapter         = "CTGCTGCCTCCCGTAGGAGT"
         String  reverse_adapter         = "AGAGTTTGATCCTGGCTCAG"
-        Int    min_length              = 1
+        Int     min_length              = 1
```

Comment:
Same as above -- even though it's nice cleanup, let's keep that to a different PR and keep this focused on documentation